CHANGELOG.md

```diff
@@ -81,7 +81,11 @@
 - Support for formatting let-else statements [#5690]
 - New config option, `single_line_let_else_max_width`, that allows users to configure the maximum length of single line `let-else` statements. `let-else` statements that otherwise meet the requirements to be formatted on a single line will have their divergent`else` block formatted over multiple lines if they exceed this length [#5684]
 
+<<<<<<< HEAD
+[#5690]: (https://github.com/rust-lang/rustfmt/pulls/5690)
+=======
 [#5690]: https://github.com/rust-lang/rustfmt/pull/5690
+>>>>>>> upstream/master
 [#5684]: https://github.com/rust-lang/rustfmt/issues/5684
```

Comment:
Ugh :vomiting_face: 

I clearly got tired of merge conflicts. Mercifully this is just noise in the changelog, so i'll clean this up later when I add the additional entries

Reply:
🤣 no worries. I know you mentioned that there were a lot of conflicts. No worries on this!